reference/performance/follower-read.md

```diff
@@ -34,6 +34,7 @@ This variable is used to set the data read mode expected by the current session.
 
 - When the value of `tidb_replica_read` is set to `leader` or an empty string, TiDB maintains its original behavior and sends all read operations to the leader replica to perform.
 - When the value of `tidb_replica_read` is set to `follower`, TiDB selects a follower replica of the Region to perform all read operations.
+- When the value of `tidb_replica_read` is set to `leader-and-follower`, TiDB can select any replicas to perform read operations.
```

Comment:
Not include learner replica

Reply:
Learners are included. In fact `follower` also includes them.

Reply:
leader-and-follower also can read learner?

Reply:
Both `leader-and-follower` and `follower` can read learner.

Reply:
How about if a learner doesn't have enough data? For example, the replica is in balancing status.

Reply:
If it's applying snapshots, new proposals will be dropped and TiDB clients will switch to a next peer.

Reply:
can we optimize it latter?  For example, peer could report its status to pd, and TiDB will not choose the peer who is applying a snapshot (or not ready).

Reply:
Good idea. Seems it's possible.